### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vbjay**
Prevent more issues like #3742
Please update with specific version info as needed.

Changes proposed in this pull request:
 - Update readme with OS support information.
 
Screenshots before and after (if PR changes UI):
NA

How did I test this code:
NA
Has been tested on (remove any that don't apply):
NA

### CODE REVIEW

**Comment by jbialobr** [comment]
I don't feel comfortable claiming the support for Linux and Mac.
I would mark the mono version as experimental.

**Comment by vbjay** [comment]
Sure. I was waiting on input like that. Why I had the question marks. I'll update later on.

**Comment by RussKie** [comment]
Agree with @jbialobr 

Having spent a bit of time working on a Mac and trying to fix Mono issues, I don't think GE in its current state is something Mac users will _really_ consider... Apple prides and preys on slick design and UX, and arguably there are git clients for Mac which provide significantly slicker UX than GE... I doubt any of those apps are built on Mono too.
https://git-scm.com/download/gui/mac

Linux users aren't as spoilt with choice as Mac users, but even they can probably get more refined UX from other clients available.
https://git-scm.com/download/gui/linux

Sadly Mono support for WinForms seems to be fundamentally broken and is incapable of providing UX comparable to what GE offers on Windows.
Even [Migel suggested to look at other platforms](http://forums.dotnetfoundation.org/t/anyone-porting-winforms-mono-to-net-core/898/2). If we want to target non Windows users we will probably need to re-engineer the app and build on GTK#...

**Comment by vbjay** [comment]
Sure. It was a quick suggestion and I put the ? to make that clear.

**Comment by vbjay** [comment]
I kind of wish we could use wpf or UWP on .net core but .net core isn't ready for that yet.

**Comment by RussKie** [comment]
I'm doubtful .NET Core will be targeting UI in near future... if ever.
Xamarin in a way is MS's xplat UI framework, but it doesn't work for linux.

**Comment by vbjay** [comment]
I am too.  I just wish they would support something where I can do the dev
once and it all works everywhere and I don't have to do web page stuff.  I
want a rich client that runs wherever .net core runs.  If they made UWP or
WPF cross platform, it would be awesome.  Xamarin forms has its place but I
want to develop for the desktop on windows and be cross platform on linux.
I think projects like this show that there is a need.

I understand that UWP is heavily dependent on DirectX and hardware
support.  I know it would take a lot of work to make UWP work on systems
that aren't windows.  I wouldn't even mind if if was Xamarin that ran on
windows and linux.  You would have to decide at project start, "Are you
sticking with windows or are you going cross platform?"

File: README.md [hunk 1]
```diff
@@ -19,6 +19,11 @@ Translation: [![Transifex](https://ds0k0en9abmn1.cloudfront.net/static/charts/im
 
 Special thanks to JetBRAINS for [Resharper](https://www.jetbrains.com/resharper/) license.
 
+## OS Support
+- Windows: 7SP1 and above
+- Linux:? and above  Mono?
+- Mac?
```
Comment (RussKie) [inline README.md]: Should we say something like

```
## OS Support
 - Windows: 7SP1 and above
 - Linux/Mac: requires [Mono 5.0](http://www.mono-project.com/download/) and above, however the support is limited due to limited support of WinForms by Mono
```

Comment (vbjay) [inline README.md]: Yes. Sounds perfect.

Comment (vbjay) [inline README.md]: Yes. Sounds perfect.

**Pull request merged**